### ISSUE TITLE
[UI] Rename "Create workbench" button to "Add workbench"

### ISSUE THREAD

**Issue by bdattoma** [opendatahub-io/opendatahub-community#68]
DS Project UI has "Add server", "Add data connection", "Add cluster storage" but we still use "Create workbench". 

I am proposing to rename it to "Add workbench" to keep consistency

**Comment by manaswinidas** [comment]
cc @kywalker-rh

**Comment by kywalker-rh** [comment]
@bdattoma If I remember correctly, we originally used "Create" for workbenches and projects because we are actually creating a new resource. We used "Add" for data connections because we aren't creating anything for it, simply adding it as something you can connect to, and we did the same things for cluster storage. Although as I think about it we are actually creating a resource for cluster storage too, so that doesn't necessarily fit with that reasoning.

@kaedward Do you have an opinion on this text consistency issue? Is it better to be very specific and consistent with the action words we use as they refer to the processes we are using on the back-end (use "Create" for creating a resource, or "Add" for adding an existing resource into an existing project) or better to be consistent throughout the UI with the action word (i.e. use "Add" for every action)?

**Comment by bdattoma** [comment]
> @bdattoma If I remember correctly, we originally used "Create" for workbenches and projects because we are actually creating a new resource. We used "Add" for data connections because we aren't creating anything for it, simply adding it as something you can connect to, and we did the same things for cluster storage. Although as I think about it we are actually creating a resource for cluster storage too, so that doesn't necessarily fit with that reasoning.
> 
> @kaedward Do you have an opinion on this text consistency issue? Is it better to be very specific and consistent with the action words we use as they refer to the processes we are using on the back-end (use "Create" for creating a resource, or "Add" for adding an existing resource into an existing project) or better to be consistent throughout the UI with the action word (i.e. use "Add" for every action)?

yes, when you Add a data connection you're creating a new secret in OpenShift, same for PV

**Comment by kaedward** [comment]
@kywalker-rh I'd like to be specific about what action is actually being done - so "create" for creating new workbenches and projects, "add" for data storage, and "create" for cluster storage (which slipped by me - sorry!)

I think it could be confusing to use "add" or "create" universally, especially as we add in new features. PF explains the difference between add + create here: https://www.patternfly.org/v4/ux-writing/terminology

**Comment by kaedward** [comment]
This makes me think of accelerator profiles - right now, we use "add", but it should be "create", since a new resource/thing is being created... Thanks for bringing this up @bdattoma, I wouldn't have noticed we were using inconsistent terminology otherwise 🙏
